frappe/core/doctype/communication/email.py

```diff
@@ -56,7 +59,7 @@ def make(doctype=None, name=None, content=None, subject=None, sent_or_received =
 	cc = list_to_str(cc) if isinstance(cc, list) else cc
 	bcc = list_to_str(bcc) if isinstance(bcc, list) else bcc
 
-	comm = frappe.get_doc({
+	comm: "Communication" = frappe.get_doc({
```

Comment:
what's the difference between 
```
	comm: "Communication" = frappe.get_doc({
```
and
```
	comm: Communication = frappe.get_doc({
```

Reply:
I tend to prefer the string types because I've come across circular imports multiple times. In some cases, you may face issues in the background but not web workers. This would go unseen until you check logs.

This just feels much safer at a point for anything that requires importing. 😅